ggshield/cmd/iac/scan/all.py

```diff
@@ -57,6 +58,8 @@ def iac_scan_all(
 ) -> Union[IaCScanResult, IaCSkipScanResult, None]:
     config: Config = ctx.obj["config"]
 
+    assert_directory_not_ignored(directory, ctx.obj["exclusion_regexes"])
```

Comment:
As `ctx.obj["exclusion_regexes"]` is now used twice, we can define a variable : 
    `exclusion_regexes = ctx.obj["exclusion_regexes"]`